india_compliance/gst_india/doctype/gstr_3b_report/gstr_3b_report.py

```
@@ -749,3 +755,297 @@ def make_json(name):
+        "gstin": 3,
+        "year": 7,
+        "month": 7,
+    }
+
+    # Section 3.1 - Tax columns
+    TAX_COLUMNS = {
+        "txval": 3,
+        "iamt": 4,
+        "camt": 5,
+        "csamt": 7,
+    }
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)